include/point/properties.hpp

```diff
@@ -96,6 +96,75 @@ struct properties<specfem::dimension::type::dim2,
   ///@}
 };
 
+template <bool UseSIMD>
+struct properties<specfem::dimension::type::dim2,
+                  specfem::element::medium_tag::elastic,
+                  specfem::element::property_tag::anisotropic, UseSIMD> {
+
+  /**
+   * @name Typedefs
+   *
+   */
+  ///@{
+  constexpr static bool is_point_properties = true;
+  using simd = specfem::datatype::simd<type_real, UseSIMD>; ///< SIMD type
+  constexpr static auto dimension = specfem::dimension::type::dim2;
+  constexpr static auto medium_tag = specfem::element::medium_tag::elastic;
+  constexpr static auto property_tag =
+      specfem::element::property_tag::anisotropic;
+  using value_type =
+      typename simd::datatype; ///< Value type to store properties
+  ///@}
+
+  /**
+   * @name Elastic constants
+   *
+   */
+  ///@{
+  value_type c11; ///< @f$ c_{11} @f$
```

Comment:
Do you not need the density for the computation of the mass matrix?

Reply:
Yes I do. Thanks! I forgot to see mass matrix.